src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -148,9 +148,11 @@ final case class RequestSession[F[_]: Monad: Trace](
       .map(_.body)
 
   def head(
-      uri: Uri
+      uri: Uri,
+      headers: Seq[Header] = Seq()
   ): F[Seq[Header]] =
     sttpRequest
+      .headers(headers: _*)
```

Comment:
note that headers from sttpRequest would be preserved if not present in argument, it makes sense by default for things like x-cdp-app, x-cdp-sdk etc, for .acceptEncoding we can still override it with headers, but if main motivation is only acceptEncoding then maybe add dedicated parameter for it?

Reply:
Maybe I can use "overrideHeaders" but that was my intention here. Just being a bit more general.